tests/cases/fourslash/completionForStringLiteralRelativeImport1.ts

```diff
@@ -51,20 +52,21 @@ for (const kind of kinds) {
 
     goTo.marker(kind + "1");
     verify.completionListContains("f1");
-    verify.completionListContains("f2");
-    verify.completionListContains("e1");
-    verify.completionListContains("folder");
-    verify.completionListContains("parentTest");
-    verify.not.completionListItemsCountIsGreaterThan(5);
+    verify.completionListContains("f3");
+    verify.completionListContains("f4");
+    verify.completionListContains("f7");
+    verify.completionListContains("d1");
+    verify.completionListContains("d2");
+    verify.not.completionListItemsCountIsGreaterThan(6);
```

Comment:
Isn't there just a "verify it is equal to 6" check? You already checked for 6 entries above, so... :-)